lib/constants.js

```diff
@@ -116,6 +116,9 @@ const PEGIN_V1_RSKT_PREFIX_HEX =  '52534b54';
 
 const MINIMUM_PEGOUT_AMOUNT_IN_RBTC = 0.0025;
 
+const FEE_PER_KB_CHANGE_PK = '6a4b49312b91e203ddfb9bc2d900ebbd46fbede46a7462e770bedcb11ad405e9';
```

Comment:
A bit nit from my side, but whenever I see `PK` I wish I could tell if it means public or private key. And the difference is quite important